kasa/__init__.py

```diff
@@ -133,6 +136,10 @@ def __getattr__(name):
     UnsupportedDeviceException = UnsupportedDeviceError
     AuthenticationException = AuthenticationError
     TimeoutException = TimeoutError
+    ConnectionType = DeviceConnection
```

Comment:
Maybe `ConnectionParameters` would be a better name as `Connection` could be confused to something that holds a connection "instance", if you know what I mean?

`EncryptType` or `EncryptionType` or `EncryptionMethod` also sounds more clear than `DeviceEncryption`, but maybe that's just me.

Reply:
Updated to `ConnectionParameters` and `EncryptionType`